content/no-code.md

```diff
@@ -106,73 +102,73 @@ The persona scenarios below are examples of accessibility issues with no-code to
-        I want to add a new block of content to my home page. The documentation of my tool says I need to drag and drop, but I cannot do that as I do not use the mouse. I tried using my screen reader’s built-in functionality to drag and drop, but still it does not work.
+        I want to add a new section to my home page. The tool documentation says to drag and drop a content block. I can't do that, because I can't use a mouse. I tried using my screen reader’s built-in functionality to drag and drop, but it doesn't work with this tool.
       </q></span></p>
     </li>
     <li>
       <p><span class="issue">Works well: </span><span><q>
-        I can use the keyboard to select the block of content I want to use and then put it wherever I want on the page. Everything is announced properly including the type of block and the location where I place it.
+        I can use the keyboard to select the content block and put it where I want on the page. Everything is announced properly, including the block type and the location where I place it.
       </q></span></p>
     </li>
   </ul>
 </div>
 
-Users who have auditory, cognitive, neurological, physical, speech, and visual disabilities need to be able to add, customize,and remove blocks of content. This includes providing clear instructions on how to use your system as well as ensuring your code works with assistive technology.
+Users who have auditory, cognitive, neurological, physical, speech, and visual disabilities need to be able to add, customize, and remove blocks of content.
 
-### The tool supports the production of accessible content
+### The tool provides accessible components and templates
 
 <p><span class="persona">Persona: </span>
-  Amahle is a hairdresser. She has learned to build a simple website to handle client appointments using a no-code tool.
+  Amahle is a hairdresser and is proud that she created her own website. Clients use her website to schedule appointments, and it's a real time saver for her.
 </p>
 
 <div class="quotes">
   <ul>
     <li>
       <p><span class="issue">Problem: </span><span><q>
-        Some of my clients cannot select the time and date when booking an appointment through my website. When I tried to make changes, the tool I am using does not help me fix these accessibility issues.
+        Some of my older clients say they cannot use the calendar widget to schedule an appointment. One says she uses large text and the buttons are cut off. Others have complained that the light gray text is too hard to read.
       </q></span></p>
     </li>
     <li>
       <p><span class="issue">Works well: </span><span><q>
-        The tool has an accessible calendar widget. It also suggests me color combinations I could use, and guides me to add headings, lists, and labels.
+        The calendar widget that the tool provides works great for all my clients, including the older clients. The tool helped me pick color combinations that are easy to see.
```

Comment:
```suggestion
        The calendar widget that the tool provides works great for all my clients, including the older clients. The tool helped me pick color combinations that are easy to see, and guides me to add headings, lists, and labels.
```

Not sure why this second part was removed. I think it fits well here and ties well to the paragraph below.

Reply:
I think we should list them in both places. Agree that does not closely relate to the persona, but it makes for an accessible output, and I think it's relevant to have hear.
Will also open a separte issue for discussion.